ssl/t1_lib.c

```diff
@@ -554,7 +556,11 @@ static int tls1_check_cert_param(SSL *s, X509 *x, int check_ee_md)
     if (!tls1_check_pkey_comp(s, pkey))
```

Comment:
should the server allow client certificates with compression here?

Reply:
It looks to me like it does? Or am I missing something? This checks that the certificate is compatible with any point formats that are present. So the only way this could go wrong is if the client says it doesn't support point compression and then sends a cert using compression. Is that an issue? If so I'd say that isn't part of this PR. There does need to be a separate PR looking at certs with compression in TLSv1.3

Reply:
I was under the impression that client certificates are checked here,
but believe now I was mistaken.